src/Backoffice/Cities/Domain/Models/City.php

```diff
@@ -40,4 +43,20 @@ public function airlines(): BelongsToMany
     {
         return $this->belongsToMany(Airline::class);
     }
+
+    /**
+     * @return HasMany<Flight>
+     */
+    public function outgoing_flights(): HasMany
```

Comment:
```suggestion
    public function outgoingFlights(): HasMany
```
Methods should be `camelCase`. Even tho the relationship is camelCame, you access it using snake_case:
```
$city->outgoing_flights;
```